### PR TITLE
Add Covid19 symptoms questionnaire

### PR DESCRIPTION
**Pull request by mpgxvii**
- Adds COVID-19 symptoms questionnaire

### CODE REVIEW

**Comment by mpgxvii** [comment]
Still currently testing the branching logic.

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": "responses['cough_wk'] ==  '1'"
+    },
+    {
+        "field_name": "covid19_mucus",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "checkbox",
```
Comment (mpgxvii) [inline questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: @afolarin I think this should be radio? (I can make the changes in REDCap)

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json [hunk 2]
```diff
@@ -0,0 +1,866 @@
...
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": "responses['fever_wk'] ==  '1'"
+    },
+    {
+        "field_name": "breath_2wk",
```
Comment (mpgxvii) [inline questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: @afolarin Should this question be before `Rate how severe your breathing has been` just like order of the 2 fever questions before this?

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json [hunk 3]
```diff
@@ -0,0 +1,866 @@
...
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": "responses['medications_intake'] ==  '1'"
+    },
+    {
+        "field_name": "covid19_meds_other",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "notes",
```
Comment (mpgxvii) [inline questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: @afolarin We don't support notes, only text box. Can I switch this to text box?

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json (hunk 2 above)
Comment (afolarin) [reply questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: Yes, thought it was

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json (hunk 3 above)
Comment (afolarin) [reply questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: Sorry I got the wrong type. Pls change

File: questionnaires/covid19_symptoms/covid19_symptoms_armt.json (hunk 1 above)
Comment (afolarin) [reply questionnaires/covid19_symptoms/covid19_symptoms_armt.json]: I wonder if it should be split as this item is really 2 questions rolled in one. Let me tak a look tonight

**Comment by afolarin** [comment]
OK I've updated the questionnaire. 
- [x] fixed ordering
- [x] changed var names so more consistent
- [x] removed some unnecessary symptoms from matrix question
- [x] put branching logic on all questions so people can skip whole questionnaire if situation has not changed and we are likely to deliver this questionnaire every day or other day

**Pull request merged**